npm_and_yarn/lib/dependabot/npm_and_yarn.rb

```diff
@@ -56,7 +66,24 @@ module NpmAndYarn
     SUB_DEP_LOCAL_PATH_TEXT = "refers to a non-existing file"
 
     # Used to identify invalid package error when package is not found in registry
-    INVALID_PACKAGE_REGEX = /Can't add "(?<package_req>.*)": invalid/
+    INVALID_PACKAGE_REGEX = /Can't add "[\w\-.]+": invalid/
```

Comment:
This is replaced for performance. We are not using `<package_req>` parameter to capture the matches. So it is replaced to just test the regex. Also for performance valid package only names are going through this test since we already have illegal package name check.